lib/core/lib/generators/ros/endpoint/endpoint_generator.rb

```diff
@@ -17,8 +17,8 @@ def create_files
       generate_policy_spec
       generate_api_doc
       generate_model
-      generate_model_specs
-      generate_and_modify_controller
+      generate_route
+      generate_controller
     end
 
     private
```

Comment:
remove `private` and all the methods that are manually invoked above will be automatically invoked so when removing `private` you can also remove the manual invocation of methods.